bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/types/CommandDescriptionBuilder.java

```diff
@@ -68,7 +77,7 @@ public CommandDescriptionBuilder withCommandOption(CommandOption commandOption)
      * @return this builder.
      */
     public CommandDescriptionBuilder withCommandOptions(List<CommandOption> commandOptions) {
-        commandOptions.forEach(co -> this.commandOptions.add(co));
```

Comment:
Currently all of our builder implementations replace the complete entity list instead of adding them to the existing list:

https://github.com/openhab/openhab-core/blob/f002a21da522b6c42a93b3cf65dce9961bdf10e9/bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/types/StateDescriptionFragmentBuilder.java#L143-L146

https://github.com/openhab/openhab-core/blob/f002a21da522b6c42a93b3cf65dce9961bdf10e9/bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/builder/ThingBuilder.java#L85-L91

Reply:
Indeed, and that's on purpose - thanks for fixing it here.